profiles/ayushlohmod.json

```diff
@@ -0,0 +1,6 @@
+{
+"image": "https://user-images.githubusercontent.com/37297939/178766012-d773a89b-8627-4ca7-bdad-5def06995121.png",
+"issueId": 1229,
+"name": "Ayush Lohmod",
+"username": "ayushlohmod"
```

Comment:
It's missing a **two-space** indentation.
```suggestion
  "image": "https://user-images.githubusercontent.com/37297939/178766012-d773a89b-8627-4ca7-bdad-5def06995121.png",
  "issueId": 1229,
  "name": "Ayush Lohmod",
  "username": "ayushlohmod"
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable